### ISSUE TITLE
.zip plugin returns "Extraction failed Too many properties to enumerate"

### ISSUE THREAD

**Issue by bigcountrycrane** [FlatFilers/flatfile-plugins#179]
Description of issue:

I created a .zip file of 3 CSVs for our "performance" repo which is meant to show off the power, speed, and experience of using Flatfile. However, when uploading the zip file I get the "Extraction failed Too many properties to enumerate" error.

How to reproduce:
1. Deploy to an Environment from https://github.com/FlatFilers/performance (follow the README)
2. Create a Space (any name, no namespace)
3. Upload the zip file here: https://drive.google.com/file/d/1wJlNV0jTXBVcgaZAMy7oQ_OWvKMXI5ZA/view?usp=sharing

**Comment by hansjhoffman** [comment]
Is there a picture or video showing what happened?

**Comment by bigcountrycrane** [comment]
https://www.loom.com/share/0475214778c64b07a22c708ca34432e0

**Comment by hansjhoffman** [comment]
This is almost certainly a "too big file size" problem.

**Comment by carlbrugger** [comment]
This is due to a `FlatfileTimeoutError: Timeout` error when the API to upload the largest of the 3 files in that zip file.

**Comment by bigcountrycrane** [comment]
@carlbrugger what's the path forward to resolve? Is this something that we need to get the Platform team involved with fixing?

**Comment by carlbrugger** [comment]
@bigcountrycrane I'm not sure which person/team should be looped into this issue. Sure, I'd be happy to see what insight the Platform team can provide.

**Comment by carlbrugger** [comment]
@bigcountrycrane so there is a 10 minute timeout on file uploads. When uploading `1e6x11.csv` (247mb) we get the timeout error long before then. I can successfully upload a file just under 200mb in size. Anything over that is consistently failing. I'm wondering if there is a file size limit somewhere.

**Comment by bigcountrycrane** [comment]
Yeah I think there must be a limit, given that we get the response back well before 10m. @sarocu @mmccooyyy tagging yall in here; I'm not sure if this has to do with the file volume or some limit on the API
